_typography.scss

```diff
@@ -0,0 +1,28 @@
+h1 {
+    font-size: 3.125rem;
+    font-weight: 700;
+}
+h2 {
+    font-size: 2.5rem;
+    font-weight: 700;
+}
+h3 {
+    font-size: 1.25rem;
+    font-weight: 500;
+}
+h4 {
```

Comment:
use `h3` instead `h4` and change font-weight by use parent element or upper level class

Reply:
it has been done